recipes/freebayes/build.sh

```diff
@@ -35,13 +43,20 @@ sed -i.bak 's/^CXX=.*$//g' Makefile
 sed -i.bak 's/^C=gcc//g' Makefile
 sed -i.bak 's/C)/CC)/' Makefile
 sed -i.bak "s:INCLUDE =:INCLUDE = -I$PREFIX/include:" Makefile
+sed -i.bak 's/configure \&\& make/configure \&\& make -e/g' Makefile
+sed -i.bak 's/export C$//g' Makefile
+sed -i.bak 's/export CC$//g' Makefile
+sed -i.bak 's/export CXX$//g' Makefile
+
 #sed -i.bak "s:LIBS =:LIBS = -L$PREFIX/lib:" Makefile	
 
 # Set exports.
 export CFLAGS="-I$PREFIX/include"
 export C_INCLUDE_PATH=$PREFIX/include
 export CPLUS_INCLUDE_PATH=${PREFIX}/include
 export LIBRARY_PATH=${PREFIX}/lib
+export CC=${CC} 
+export CXX=${CXX} 
```

Comment:
They're already exported, you should be able to remove these two lines.

Reply:
Thanks!